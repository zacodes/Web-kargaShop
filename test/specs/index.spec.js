import { mount, shallowMount } from '@vue/test-utils'
import index from "../../pages/index.vue"


describe('index.vue', () => {

    it('test1-id', () => {
        const wrapper = shallowMount(index)
        expect(wrapper.find('#vt').text()).toContain('Vitrindekiler');
    });

    it('index.vue Bileşenimizin class özelliğini kontrol edilmesi', () => {
        const wrapper = mount(index)
        expect(wrapper.classes()).toContain('container');
    });

    it("yazi", () => {
        const wrapper = mount(index)
        expect(wrapper.text()).toContain("Vitrindekiler");
    });

    it('Snapshot özelliği', () => {
        const wrapper = mount(index)
        expect(wrapper.html()).toMatchSnapshot()
    });

    it('has data', () => {
        const wrapper = mount(index)
    expect(typeof index.data).toBe('function')
    });

})

    /*
describe('index.vue', () => {
    it('test1-', () => {
        const wrapper = shallowMount(index)
        expect(wrapper.find('#vitrinyazi').text()).equal('Vitrindekiler');
    });

    it('test2- class özelliği kontrol', () => {
        const wrapper = mount(index)
        expect(wrapper.classes()).toContain('container');
    });

    it('test3- class özelliği kontrol', () => {
        const wrapper = mount(index)
        expect(wrapper.classes()).toContain('card-price font-weight-bold');
    });

    it('test4- snapshot özelliği', () => {
        const wrapper = mount(index)
        expect(wrapper.html()).toMatchSnapshot();
    });


    })

    */