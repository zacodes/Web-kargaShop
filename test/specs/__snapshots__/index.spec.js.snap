// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.vue Snapshot özelliği 1`] = `
<div id="Home" class="container">
  <b-carousel id="carousel-1" interval="4000" controls="" indicators="" background="#fff" img-width="100%" style="text-shadow: 1px 1px 2px #333;">
    <b-carousel-slide img-src="~/assets/image/slider_1.jpg"></b-carousel-slide>
    <b-carousel-slide img-src="~/assets/image/slider_2.jpg" href=""></b-carousel-slide>
    <b-carousel-slide img-src="~/assets/image/slider_3.jpg" href=""></b-carousel-slide>
    <b-carousel-slide img-src="~/assets/image/slider_4.jpg" href=""></b-carousel-slide>
    <b-carousel-slide img-src="~/assets/image/slider_5.jpg" href=""></b-carousel-slide>
  </b-carousel>
  <p id="vitrinyazi" class="text-center text-uppercase mt-5 text-lg pb-0 mb-0">
    Vitrindekiler
  </p>
  <hr class="head-border mx-auto mt-1">
  <div class="row">
    <div class="col-3 product-card-container">
      <b-link to="/product">
        <b-card title="KANAGAWA -UNISEX TSHIRT" img-src="_nuxt/assets/image/product1.jpg" img-alt="KANAGAWA -UNISEX TSHIRT" img-top="" tag="article" class="mb-2" style="max-width: 20rem;">
          <b-card-text>
            KANAGAWA -UNISEX TSHIRT
          </b-card-text>
          <p class="card-price font-weight-bold">
            34.9 TL
          </p>
        </b-card>
      </b-link>
    </div>
  </div>
</div>
`;
